homeassistant/components/opengarage/cover.py

```diff
@@ -87,7 +90,7 @@ def _update_attr(self) -> None:
         status = self.coordinator.data
 
         self._attr_name = status["name"]
-        state = STATES_MAP.get(status.get("door"))
+        state = STATES_MAP.get(status.get("door"))  # type: ignore[arg-type]
```

Comment:
```suggestion
        state = STATES_MAP.get(status["door"])
```
Don't know if `door` is a required field for `status`. If it is, the code above would work.

Reply:
Let's go with the ignore for now.